esmvalcore/cmor/_fixes/cmip5/bcc_csm1_1.py

```diff
@@ -1,9 +1,65 @@
 """Fixes for bcc-csm1-1."""
+import iris
 import numpy as np
 from scipy.interpolate import InterpolatedUnivariateSpline
 from scipy.ndimage import map_coordinates
 
 from ..fix import Fix
+from ..shared import fix_bounds
+
+
+class Cl(Fix):
```

Comment:
Since this is a fix that is applied to very many models and is not at all specific to bcc-csm, maybe it's a good idea to put it into a file `common.py` and to apply it to bcc-csm1-1 in the same way as all the other models?

Reply:
Good call, will do that!